screens/OrderFormIV.js

```diff
@@ -44,143 +44,143 @@ class OrderFormIV extends React.Component {
 
     return (
       <KeyboardAvoidingView behavior="padding" enabled style={[{}]}>
-      <ScrollView scrollIndicatorInsets={{ right: 1 }}>
-        <View style={styles.fullWidthScreen}>
-          <ScreenHeader backNavigation={true}
-                        parentFullScreen={true}
-                        title={product.name}
-          />
-
-          {product.productOptions != null && product.productOptions.length > 0 && (
-            <View style={[styles.tableRowContainerWithBorder]}>
-              <Text style={styles.tableCellText}>
-                {t('productOptions')}
-              </Text>
-            </View>
-            )}
+        <ScrollView scrollIndicatorInsets={{ right: 1 }}>
```

Comment:
Not sure what was changed in this screen. It seems to be my change from pull request 76. how did you do the rebase operations?